app/assets/stylesheets/media.scss

```diff
@@ -275,9 +275,8 @@
     flex-shrink: 0;
   }
 
-  // First selector is for embargo. Last selector is other auth restrictions
-  .authLinkWrapper + .media-component-body, [data-controller="auth-restriction"]:has(.authLinkWrapper:not([hidden])) + .media-component-body {
-    max-height: calc(100vh - 88px); // Scroll height for metadata when the auth wrapper is displaying
+  [data-controller="auth-restriction"]:not([hidden]) + .media-component-body {
+    max-height: calc(100vh - 100px); // Scroll height for metadata when the auth wrapper is displaying
```

Comment:
if i left it at `88px`, the bottom of the control bar was still a bit cut off, e.g.

<img width="1558" alt="Screen Shot 2023-11-29 at 10 00 02 PM" src="https://github.com/sul-dlss/sul-embed/assets/7741604/3e305b9c-8a5f-4e51-8fb2-60cd9feb1c08">

vs after this fix:
<img width="1564" alt="Screen Shot 2023-11-29 at 10 00 55 PM" src="https://github.com/sul-dlss/sul-embed/assets/7741604/c9a3f7a7-fb68-453b-b040-c8c524a4c316">